data_architect_misc/data_transformer/transform_functions/latam_paraguay_transform_functions.py

```diff
@@ -0,0 +1,42 @@
+"""This is the subclass of Transform function for Paraguay (LATAM division).
+
+We will define transform functions specific to Paraguay here.
+
+Author: Maicol Contreras
+Last Modified: September 9, 2020
+"""
+
+import pandas as pd
+
+from constants import comp_harm_constants
+from transform_functions.common_comp_harm_transform_functions import CommonCompHarmTransformFunctions
+
+
+class LatamParaguayTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    PARAGUAY_SPECIFIC_CATEGORY_MAPPINGS = {
+        "(?i).*TRAPEADOR.*": "Home Care",
```

Comment:
I think I've seen 'Trapeador' many times in other LATAM countries too. If that's the case, since it is not 'Other' category, shouldn't we be adding this to the constants.py file?

Reply:
Yes sire, I'm using the same transform module for creating this file. for that reason, the same category is appear in several transform module. but I put this one only for keeping in mid or as an example